docs/search/criteria_reference/isvirtual_criterion.md

```diff
@@ -0,0 +1,45 @@
+# IsVirtual Criterion
+
+The `IsVirtual` Search Criterion searches for virtual products. 
+
+## Arguments
+
+- (optional) `value` - bool representing whether to search for virtual (default `true`)
+or physical (`false`) products.
+
+## Limitations
+
+The `IsVirtual` Criterion is available in Solr or Elasticsearch engines.
+
```

Comment:
Criterion is available in all supported search engines so there are no limitations. Based on other described criterions for products, Limitations section should be added when at least one of engine is not supported.